src/app/questions/questionhelper.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { QuestionHelper } from './questionhelper';
+import { mockQuestionsJson } from '../jsonresponse.mock';
+
+const mockAnswer1 = [
+  { 'optionId': '015', 'checked': true },
+  { 'optionId': '012', 'checked': true },
+  { 'optionId': '011', 'checked': true },
+  { 'optionId': '014', 'checked': true },
+  { 'optionId': '002', 'checked': true },
+  { 'optionId': '013', 'checked': true },
+  { 'optionId': '003', 'checked': true }
+];
+const mockAnswer2 = [{ 'optionId': '011', 'checked': true }];
+const mockAnswer3 = [{ 'optionId': '011', 'checked': false }];
+
+describe('QuestionHelper', () => {
+  beforeEach(() => {
+  });
+
```

Comment:
Please make your tests more explicit, with one assertion per test. That way they also serve as documentation and help you write more testable code.
For example "It should deduct 1 point for a wrong answer" or "It should return 0 when the amount of points is negative"